subgraph/schema.graphql

```diff
@@ -46,12 +46,15 @@ type Extend @entity(immutable: true) {
   end: BigInt! # uint64
 }
 
-type ActiveSubscription @entity {
+type Subscription @entity {
   id: Bytes!
   user: User!
   start: BigInt! # uint64
   end: BigInt! # uint64
   rate: BigInt! # uint128
+  # The subgraph preserves `end` timestamp of cancelled subscriptions to enable showing it in UI
+  # Even if not cancelled an expired subscription is inactive.
+  cancelled: Boolean!
```

Comment:
Just curious, but why `cancelled` over like `active`? Idk why `active` feels more appropriate, but it does. But just curious your thoughts

Reply:
I wrote about it in the comment. `active = true` would suggest that a subscription can be used even if `end` is in the past.